metrics.go

```diff
@@ -19,14 +19,17 @@ func registerVersionMetric(version string) {
 	m.Set(1)
 }
 
+// Duration histograms  use fixed definition here, as we don't want to break existing buckets if we need to add more.
+var defaultDurationHistogramBuckets = []float64{0.05, 0.1, 0.2, 0.5, 1, 2, 5, 10, 30, 60, 120, 240, 480, 960, 1920}
```

Comment:
:warning: Boxo uses `0.25` but we had `0.2` bucket already, so I kept `0.2` to maintain continuity with historical histogram.

@2color lmk if you feel we should change it to `0.25` to have the same buckets everywhere.

Reply:
I would probably use the same buckets as Boxo for easier aggregation, since Rainbow exposes all the metrics coming from boxo too.

Reply:
Generally speaking, metrics aren't supposed to be kept for that long (not actually sure what's our retention), so any challenges that may arise for past data (though these would be minor since you can still aggregate, just with less granularity) would be minimal.

Reply:
Final point: from my own observations, the lower buckets don't matter so much since requests that typically hit Rainbow take in the order of seconds. 

```suggestion
var defaultDurationHistogramBuckets = []float64{0.05, 0.1, 0.25, 0.5, 1, 2, 5, 10, 30, 60, 120, 240, 480, 960, 1920}
```